### ISSUE TITLE
Accept comments in Argument files

### ISSUE THREAD

**Issue by MisterY** [simonmichael/hledger#1027]
Comments are generally useful, including the argument files. These files can serve as batch files, encapsulating various options. I.e. balance sheet for a group of accounts.
This flexibility will lead me to create multiple files with varying options and, hence, it would be useful to have some explanations about what the files are for, comment-out certain lines at times, etc.

It would be good to simply ignore commented lines, using ';' (or other standard comment characters).

**Comment by simonmichael** [comment]
I agree but I think it's not supported by the [cmdargs](http://hackage.haskell.org/package/cmdargs) lib which we use. Perhaps you can open this issue there ?
